### PR TITLE
feat: add Contact component and ContactBlock

### PR DESCRIPTION
**Pull request by edvicaty**
ticket: GAT-41 https://app.clickup.com/t/gjzdb9
features:
- add Contact component
image
- add ContactBlock
image

### CODE REVIEW

File: packages/gatsby-theme/src/blocks/ContactBlock/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import PropTypes from "prop-types"
+import Contact from "../../components/Contact"
```
Comment (luisduenas) [inline packages/gatsby-theme/src/blocks/ContactBlock/index.js]: Please add the Contact component to the [component index.js file](https://github.com/octahedroid/drupal-gatsby-theme/blob/master/packages/gatsby-theme/src/components/index.js) and update this import (see Button component example)
```suggestion
import { Contact } from "ui"
```

File: packages/gatsby-theme/src/components/Contact/ContactHeader.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import PropTypes from "prop-types"
+import RichText from "../RichText"
```
Comment (luisduenas) [inline packages/gatsby-theme/src/components/Contact/ContactHeader.js]: Please add RichText component to the [component index file](https://github.com/octahedroid/drupal-gatsby-theme/blob/master/packages/gatsby-theme/src/components/index.js) so you can update this import 
```suggestion
import { RichText } from "ui"
```

File: packages/gatsby-theme/src/components/Contact/ContactItem.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import PropTypes from "prop-types"
+import { Box } from "theme-ui"
+import Icon from "../Icon"
+import RichText from "../RichText"
+import Heading from "../Heading"
+
```
Comment (luisduenas) [inline packages/gatsby-theme/src/components/Contact/ContactItem.js:4]: Same as the others, please take this opportunity to add this components to the [component index file](https://github.com/octahedroid/drupal-gatsby-theme/blob/master/packages/gatsby-theme/src/components/index.js) and update this imports
```suggestion
import { Icon } from "ui"
import { RichText } from "ui"
import { Heading } from "ui"
```

File: packages/gatsby-theme/src/components/Contact/index.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import Contact from "./Contact"
+import ContactGrid from "./ContactGrid"
+import ContactHeader from "./ContactHeader"
+
+Contact.Grid = ContactGrid
+Contact.Header = ContactHeader
+
+export default Contact
```
Comment (luisduenas) [inline packages/gatsby-theme/src/components/Contact/index.js:8]: Please add this component to the [component index file](https://github.com/octahedroid/drupal-gatsby-theme/blob/master/packages/gatsby-theme/src/components/index.js)

File: packages/gatsby-theme/src/blocks/ContactBlock/ContactBlock.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+<Preview>
+  <Story
+    name='Default'
+    args={{
+      heading:
+        "Tell us about your project. And we will contact you promptly to discuss next steps.",
+      contactInfo,
+    }}
+    argTypes={contactArgTypes}
+  >
```
Comment (dinartecc) [inline packages/gatsby-theme/src/blocks/ContactBlock/ContactBlock.stories.mdx]: The argTypes object should be passed to the Meta tag instead. This allows the argTypes object to be applied to all the stories. In this case it is a single story, but we should change this to standardize future implementations.
```suggestion
<Meta title='blocks/ContactBlock' component={ContactBlock} argTypes={contactArgTypes} />

# ContactBlock

You can use it to add a Contact block

## Props

<Props of={ContactBlock} />

export const ContactBlockTemplate = (args) => <ContactBlock {...args} />

### Default ContactBlock

<Preview>
  <Story
    name='Default'
    args={{
      heading:
        "Tell us about your project. And we will contact you promptly to discuss next steps.",
      contactInfo,
    }}
  >
```

File: packages/gatsby-theme/src/components/Contact/index.js (hunk 4 above)
Comment (edvicaty) [reply packages/gatsby-theme/src/components/Contact/index.js:8]: done @luisduenas

File: packages/gatsby-theme/src/blocks/ContactBlock/ContactBlock.stories.mdx (hunk 5 above)
Comment (edvicaty) [reply packages/gatsby-theme/src/blocks/ContactBlock/ContactBlock.stories.mdx]: done @dinartecc

**Review by luisduenas — APPROVED** [review]
LGTM

**Pull request merged**